frontend/app/screens/welcome-screen/welcome-screen.tsx

```diff
@@ -1,39 +1,77 @@
 import React from "react";
-import { View, ViewStyle, TextStyle, SafeAreaView } from "react-native";
+import { View, ViewStyle, TextStyle, SafeAreaView, Image, StyleSheet, Dimensions } from "react-native";
 import { useNavigation } from "@react-navigation/native";
 import { observer } from "mobx-react-lite";
 import { Button, Header, Screen, Text } from "../../components";
 import { color, spacing, typography } from "../../theme";
 
-const FULL: ViewStyle = { flex: 1 };
+const darkAqua = "#008080";
+const aqua = "#46BFAC";
+const windowWidth = Dimensions.get('window').width;
+const windowHeight = Dimensions.get('window').height;
+const lightseagreen = "#616F6C";
+const background = "#fff";
+
+const styles = StyleSheet.create({
+  heading: {
+    fontSize: 28,
+    textAlign: 'center',
+    color: lightseagreen
+  }, 
+  subheading: {
+    fontSize: 17,
+    textAlign: 'center',
+    color: lightseagreen, 
+    marginTop: spacing[3], 
+    fontStyle: 'italic'
+  },
+  image: {
+    width: windowWidth,
+    height: windowWidth*1.2,
```

Comment:
This is too big on my phone, (I think mine is Pixel 2). Could you try this?

```suggestion
    height: windowHeight - 220,
```

Reply:
:o how is your screen height not a lot bigger than 1.2* the width?

Reply:
fixed to make it a square